packages/components/src/internal/components/gridbar/ExportModal.tsx

```diff
@@ -15,7 +15,14 @@ const DEFAULT_TITLE = 'Select the Tabs to Export';
 
 export const ExportModal: FC<ExportModalProperties> = memo(props => {
     const { queryModels, tabOrder, onClose, onExport, canExport, title = DEFAULT_TITLE } = props;
-    const [selected, setSelected] = useState<Set<string>>(() => new Set(tabOrder));
+    const [selected, setSelected] = useState<Set<string>>(() => {
+        let selected = new Set<string>();
+        tabOrder.forEach((modelId) => {
+            if (queryModels[modelId].rowCount > 0)
+                selected = selected.add(modelId);
+        });
+        return selected;
```

Comment:
I know this wasn't the ask for this issue, but it may seem somewhat random that some tabs are selected and some are not since there's no info in the modal about the row count. Perhaps we should add the row count after the model name as we do for the tab labels themselves.

Reply:
added.